domaintools/api.py

```diff
@@ -407,8 +404,6 @@ def iris_detect_watched_domains(self, monitor_id=None, escalation_types=None, tl
                                 order=None, include_domain_data=False, offset=0, limit=None, preview=None, **kwargs):
         """Returns back a list of watched domains in Iris Detect based on the provided filters.
         """
-        if discovered_since and changed_since:
-            raise ValueError('Please only use one of discovered_since or changed_since in your query.')
```

Comment:
I changed my mind on this because if the user really wants to do this we might as well let them.